webserver/server/app/models/transaction/sqlHistoryForAddresses.sql

```diff
@@ -69,11 +69,11 @@ WITH
                 "Block".era as "era!",
                 "Block".height as "height!",
                 "TransactionMetadata".payload AS metadata,
-                json_agg(DISTINCT "Address".PAYLOAD) input_addresses
+                array_agg("TransactionOutput".PAYLOAD) input_utxo
```

Comment:
do we care about the order of these? I think not since inputs are a set, but asking just in case, since there is no way to match these with the inputs in the tx body otherwise.

Reply:
yeah we probably should keep them ordered. Although strictly speaking Cardano doesn't enforce an order, this was definitely a mistake in the Cardano infrastructure since it means matching inputs for Plutus is really expensive since you need to do a O(n) scan over all inputs instead of an O(1) lookup for the index you know and we probably shouldn't make the same mistake